src/System.Private.ServiceModel/tests/Common/Infrastructure/FrameworkID.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Infrastructure.Common
+{
+    [Flags]
```

Comment:
Both FrameworkID and OSID are flags because they are bit masks.  They are bit masks to allow combinations like Any.  And you can combine them in attribute declarations to include or exclude certain frameworks like this:

```
    [WcfFact]
    [Issue(999, Framework = FrameworkID.Any ^ FrameworkID.NetFramework)]
    public static void Skip_Issue_All_Frameworks_Except_Full_Framework()
    {
    }

    [WcfFact]
    [Issue(999, Framework = FrameworkID.NetCore | FrameworkID.NetFramework)]
    public static void Skip_Issue_On_NetCore_And_Full_Framework()
    {
    }
```